apps/directory/src/functions/viewmodelMapper.js

```diff
@@ -357,6 +375,14 @@ export const collectionReportInformation = (collection) => {
   return collectionReport;
 };
 
+export const studyReportInformation = (study) => {
```

Comment:
Rename to getStudyReportInformation (or something like it), currently it read like a variable name.

Reply:
done